src/main/java/in/projecteka/consentmanager/common/ServiceCaller.java

```diff
@@ -0,0 +1,13 @@
+package in.projecteka.consentmanager.common;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+
+@Getter
+@Value
+@Builder
+public class ServiceCaller {
+    String clientId;
+    Role role;
+}
```

Comment:
List.Of Role is always a better option. and List could be empty for older HIU/HIP which is calling CM directly.